### ISSUE TITLE
webpage: display the applied settings in the html file.

### ISSUE THREAD

**Issue by ksauraj** [ksauraj/jee_counsellor#13]
We can add feature to display the applied settings like what Institutes Type, Course Type, Quota, Rank etc was applied in following filtered data set. 

**Objective** - This would make sharing of html file more useful as it will make sense what filters were applied during creating this filtered data.

For example -

```bash
Rank - 12424
Gender - Gender-Neutral
Seat Type - Open
Quota - AI, HS
Institutes - ALL
Institute Type - NITs
Round - 6
Year - 2023
Counseling Type - JOSAA
```

> This needs to be added in webpage ( find a suitable place to add )

**Comment by praptisharma28** [comment]
Assign me @ksauraj under iwoc24, will add!

**Comment by ksauraj** [comment]
> Assign me @ksauraj under iwoc24, will add!

Go ahead

**Comment by ksauraj** [comment]
He @praptisharma28 are you still working on this issue? There might be a conflict due to this [issue](https://github.com/ksauraj/jee_counsellor/issues/11). So just wanted to know about the progress so that i can be prepared for such conflict.

**Comment by praptisharma28** [comment]
Yeah I am working, I will update you as soon as I get to show you something. @ksauraj

**Comment by shaluchan** [comment]
please assign this issue to me....

**Comment by ksauraj** [comment]
Sure, I'm assigning this to you

**Comment by sparshsinghal885** [comment]
please assign this to me
